### PR TITLE
Hall of Fame Design

### PR DESCRIPTION
**Pull request by tonyxj**
image

Hi guys, @psybuzz @nikilselvam 
I've pushed up the new design with some dummy html. The conversation classes are most the same as the Home page. This PR is the basic foundation for us to build upon for the TOC. In the spirit of keeping tasks short, I'll be adding the Convershapes in another PR. The same regarding the Stats panel, it can be addressed in another PR. However I was thinking perhaps surfacing two numbers for Stats, 1: total conversations participating in, 2: number of conversations saved to HOF. 

Is it possible to surface those numbers? What are the time/effort needed to surface them? Having those in mind will help to determine if we can/should build stats right now. 

Thanks guys!

### CODE REVIEW

File: public/sass/hof.scss [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+.wrapper {
+	h2 {
+		margin-bottom: 2rem;
+	}
+	.convo {
+		margin-top: 0.7rem;
+		padding: 1rem 1.3rem;
+		transition: background-color 0.35s;
+
+		h3	{
```
Comment (nikilselvam) [inline public/sass/hof.scss]: We can reduce the extra spacing here between `h3` and `{`

File: views/halloffame.html [hunk 2]
```diff
@@ -25,13 +26,51 @@
...
+						</div>
 					</div>
 				</div>
 
 			</div>
 		</section>
+		<div class="right-container">
+			<section class="hof-helper right-side">
+				<h2>What are <br><span>Golden<br> Conversations?</span></h2>
+				<p>Conversations that <span class="hof-helper-emphasis">all</span> participants have added to the Hall of Fame are <span class="gold-label">golden</span>. Conversations with <span class="hof-helper-emphasis">one or more</span> participants adding to the HOF are <span class="goldplated-label">gold plated</span>. For conversations only you’ve added to the HOF, there is no gold. So Ask the other participants to add them so they too can become golden!</p>
```
Comment (nikilselvam) [inline views/halloffame.html]: I would reword this a little bit. I think the second sentence should be, `Conversations that one or more participants have added to the HOF are gold plated.` This adds some parallelism with the first sentence. In the third sentence, I would add `that` between `For conversations` and `only you`.

I'd also reword the last sentence to focus on having other participants archive conversations: `So ask the other participants to archive these conversations as well so that they too can become golden!` Let me know what you think!

**Comment by nikilselvam** [comment]
This looks good! The color really makes the page stand out. I have a couple of comments related to spacing and wording, but otherwise this LGTM.

With regards to your question about stats @tonyxj , those stats should be relatively pretty easy to surface. Each user has a collection of conversations that they are participating in in the database, and each of these conversations has a boolean indicating whether the user has archived it or not. All we  need to do is to iterate through all of these conversations in order to determine how many conversations have been archived in general and how many conversations the user has in total. I think we can do this with a few calls and some processing in the backend! What do you think @psybuzz ?

**Comment by psybuzz** [comment]
@nikilselvam In regards to stats, I think we can write stats logic on the front-end to do the things you mentioned.  In home.js, around line 100, we should be getting all of the user's conversations, so we could simply count them and filter to see how many are in the HoF.

File: views/halloffame.html (hunk 2 above)
Comment (psybuzz) [inline views/halloffame.html]: I like the span labels.  What do you guys think about using the future tense?  As in, `Convos that all participants have added will be gold plated`?

**Comment by psybuzz** [comment]
I noticed that when the browser width is small, the right-pane gets reaallly close to the left pane.  Upon inspection, it seems that the right pane as a percentage margin-left.  This might not be related to the changes you made, but is this intentional?
image

**Comment by psybuzz** [comment]
Nice changes, @tonyxj !  I had a comment about the copy/writing, but as long as it's concise and readable, I'm down for it.  For the stats, it will likely require a separate PR to expose that data, so for now, could you work with some hard-coded, fake numbers?  Other than that, LGTM.

File: public/sass/hof.scss (hunk 1 above)
Comment (tonyxj) [inline public/sass/hof.scss]: Nice catch. Fixing in the next commit.

File: views/halloffame.html (hunk 2 above)
Comment (tonyxj) [inline views/halloffame.html]: I think both of you guys have some great feedback here. I would combine your thoughts and make changes accordingly. For sure the future tense might be better, especially when you first come in to the HOF and sees nothing but the helper text.

**Comment by tonyxj** [comment]
@psybuzz Nice catch regarding the browser width! That's definitely something I need to address. For now I've inherited the home.scss where this behaviour is also showing. However, adjusting the screen any smaller than that will trigger the mobile view which fixes the margin problem. I'll be tweaking this along with the placeholder number stats in the next PR. Thanks guys!
